src/fragments/ui/auth/flutter/authenticator.mdx

```diff
@@ -159,6 +181,110 @@ return Authenticator(
 ![AmplifyTheme](/images/flutterAuthenticator/flutter_authenticator_amplify_theme.png)
 ![AmplifyTheme Dark](/images/flutterAuthenticator/flutter_authenticator_amplify_theme_dark.png)
 
+## Full UI Customization
+
+In addition to customizing form fields and theming, you can also fully customize the UI for one or more of the authentication steps (Sign In, Sign Up, Confirm Sign Up, Forgot Password). This can be used for simple use cases, such as adding a logo to the sign up and sign in screen, as well as far more complex use cases.
+
+This is done by providing a builder to the Authenticator's `authenticatorBuilder` value. If you are familiar with widgets like `StreamBuilder`, `FutureBuilder`, or `LayoutBuilder`, this may feel familiar. `authenticatorBuilder` is a builder method that takes arguments for the current `BuildContext` and `AuthenticatorState`, which is a representation of the state being managed by the authenticator.
+
+Below is an example of how you can use `authenticatorBuilder` to add a logo to both the sign up and sign in screen, as well as change the layout to remove the default tab structure.
+
+This example uses the `SignInForm()` and `SignUpForm()` widgets from the amplify_authenticator package. In addition to the available prebuilt forms, form fields (ex: `SignInFormField.username()` or `SignUpFormField.email()`) and form buttons (ex: `SignInButton()`) are also available. All of the prebuilt widgets are fully integrated into the authenticator's state. 
```

Comment:
Do we need both of these examples? It makes the code sample kind of verbose.

Reply:
I could change it to the following. I wanted to include a could examples since there is no list of all the available components (which I think we should include in the docs before GA, but I did think was needed now).

```suggestion
This example uses the `SignInForm()` and `SignUpForm()` widgets from the amplify_authenticator package. In addition to the available prebuilt forms, form fields (ex: `SignInFormField.username()`) and form buttons (ex: `SignInButton()`) are also available. All of the prebuilt widgets are fully integrated into the authenticator's state. 
```